### ISSUE TITLE
[Feature Request]: Add Plex KR and JP to Storybook

### ISSUE THREAD

**Issue by jeffchew** [carbon-design-system/carbon#15297]
### The problem

Plex Korean and Japanese have been available for awhile, though are currently not listed in the React Storybook navigation:

https://react.carbondesignsystem.com/?path=/docs/elements-ibm-plex--overview

### The solution

Add Korean and Japanese to the list of available Plex fonts in storybook.

### Business priority

Low Priority = release date is not dependent on fix or not upcoming

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)

**Comment by tay1orjones** [comment]
Great catch! This is a complicated one. Plex JP and KR aren't listed in storybook because right now they're actually not available directly through Carbon. We initially documented this here:

https://github.com/carbon-design-system/carbon/blob/d9121f17b1dd935a5734c6eae10af275f72f6dc7/packages/styles/scss/fonts/README.md?plain=1#L7-L18

If consumers want to use these, they have to include the assets themselves (via the akamai CDN or self-hosted) and craft the `@font-face` at-rule(s) themselves.

These were left out during the [initial implementation](https://github.com/carbon-design-system/carbon/pull/10205) due to three reasons:

* At the time, JP was not included in the current Plex beta
* These fonts support hinted and unhinted variants which will complicate an already pretty complicated resolver logic.
* The font files and resulting package can be HUGE

So this will be a larger issue to add support for these in addition to surfacing them in the storybook. May also be dependent on #11661

**Comment by jeffchew** [comment]
Thanks for the detailed explanation @tay1orjones !

^^ FYI @mjabbink
